server/Gemfile

```diff
@@ -22,8 +22,10 @@ gem "activestorage-validator"
 gem "pundit"
 gem "active_model_serializers"
 gem "uuid"
+gem "rails-i18n", "~> 7.0", ">= 7.0.1"
```

Comment:
as I remember this gem had to be removed to resolve bug with translations switching...